docs/modules/ROOT/pages/advanced-how-tos/installing/index.adoc

```diff
@@ -1 +1,11 @@
-= Installing {ProductName}
\ No newline at end of file
+= Installing {ProductName}
+
+{ProductName} offers a comprehensive solution that fortifies your software supply chain against emerging threats. By providing detailed insights into the composition and construction of your software, {ProductName} enables proactive identification and mitigation of vulnerabilities.
+
+Before tapping into the vast array of benefits offered by {ProductName}, the initial step involves its installation. Users have two options for onboarding applications to {ProductName}:
+
+. *Using the {ProductName} UI*: This option provides a more streamlined user experience but requires using Quay.io for the image registry and additional initial setup steps.
+
+. *Using Kubernetes manifests*: This option supports any image registry and involves fewer initial setup steps.
+
+For detailed instructions, refer to link:https://github.com/{ProductName}-ci/{ProductName}-ci/blob/main/README.md[{ProductName}-CI] guide.
```

Comment:
Can we have it without the main title inside the page? That title is at the top of the page, adding it to the link adds another point of failure in case that the title changes. If we drop it, I think the link will just point to the same place, but with lesser risk of getting broken.
```suggestion
For detailed instructions, refer to link:https://github.com/{ProductName}-ci/{ProductName}-ci/blob/main/README.md guide.
```